Contributing.md

```diff
@@ -0,0 +1,16 @@
+
+### Committing a change
+
+#### Development
+
+When opening a pull request, add your changes to the "[Unreleased]" section of the [changelog](https://github.com/zendesk/active_record_shards/blob/master/Changelog.md).
+
+#### Release
+
+After merging your changes into master, cut a tag and push it immediately:
+
+1.  Update the version by `bundle exec rake bump:patch`, `bundle exec rake bump:minor` or `bundle exec rake bump:major`.
```

Comment:
Do people really use `rake bump`? I always just manually change the version file.

Reply:
Should we remove `bump` from the gemspec then?

Reply:
I use `bundle exec bump` :) But yeah, in most repositories (like here I assume) rake just changes the version file, so seems fine I think manually updating it as part of the release steps.

Reply:
Just realized there is no version.rb file in the gem oO Updating manually still seems fine though

Reply:
updated to tell people to just modify the gemspec file